Automattic/wp-calypso
# Remove code related to the Videopress flows

Related to https://github.com/Automattic/dotcom-forge/issues/9333

## Proposed Changes

* On https://github.com/Automattic/wp-calypso/pull/95751 I removed the code related to the Videopress TV and Videopress TV purchase flows, and on this PR I'm tackling the Videopress flow.
* I also found some old code related to [the now unused `/start/videopress` flow](https://github.com/Automattic/wp-calypso/pull/66411), so I removed it as well

## Testing Instructions



* Open the live preview
* Go to `/setup/videopress`
* Check that you are redirected to `/start/domains`
* Repeat the previous steps with `/start/videopress`
* Check that `/start/videopress-account/user` still works:

|Logged in|Logged out|
|---|---|
|![image](https://github.com/user-attachments/assets/b077f8b1-a3f2-41bf-a1d3-ba9924f870d6)|![image](https://github.com/user-attachments/assets/1aa8825e-0dfe-4bdf-ae73-668e5860a3ea)|

## Pre-merge Checklist
